File: ncDados/dEfdArquivo.vb
Namespace nsEFD

  Public Class dEfdArquivo

    Private _versaoLeiaute As String
    Private _finalidadeArquivo As String
    Private _perfilArquivoFiscal As String

    Public Property versaoLeiaute() As String
      Get
        Return _versaoLeiaute
      End Get
      Set(ByVal value As String)
        _versaoLeiaute = value
      End Set
    End Property

    Public Property finalidadeArquivo() As String
      Get
        Return _finalidadeArquivo
      End Get
      Set(ByVal value As String)
        _finalidadeArquivo = value
      End Set
    End Property

    Public Property perfilArquivoFiscal() As String
      Get
        Return _perfilArquivoFiscal
      End Get
      Set(ByVal value As String)
        _perfilArquivoFiscal = value
      End Set
    End Property

  End Class

End Namespace

File: ncModelos/dEfdArquivo.cs
using System;

namespace ncModelos
{
    public class dEfdArquivo
    {
        private string _versaoLeiaute;
        private string _finalidadeArquivo;
        private string _perfilArquivoFiscal;

        public string versaoLeiaute
        {
            get => _versaoLeiaute;
            set => _versaoLeiaute = value;
        }

        public string finalidadeArquivo
        {
            get => _finalidadeArquivo;
            set => _finalidadeArquivo = value;
        }

        public string perfilArquivoFiscal
        {
            get => _perfilArquivoFiscal;
            set => _perfilArquivoFiscal = value;
        }
    }

}
